apps/api/src/app/subscribers/usecases/chat-oauth/chat-oauth.usecase.ts

```diff
@@ -38,13 +76,45 @@ export class ChatOauth {
       );
     }
 
+    if (integration.credentials == null) {
```

Comment:
```suggestion
    if (!integration.credentials) {
```

Is there a reason not to do it this way? Feels cleaner